wsi/drm/drmdisplay.cpp

```diff
@@ -537,15 +537,9 @@ bool DrmDisplay::GetFence(drmModeAtomicReqPtr property_set,
   return true;
 }
 
-void DrmDisplay::Disable(const DisplayPlaneStateList &composition_planes) {
+void DrmDisplay::Disable() {
   IHOTPLUGEVENTTRACE("Disable: Disabling Display: %p", this);
 
-  for (const DisplayPlaneState &comp_plane : composition_planes) {
-    DrmPlane *plane = static_cast<DrmPlane *>(comp_plane.plane());
```

Comment:
Good to reset the state neverthless, so I think we want to do this.

Reply:
After this disable call the previous_plane_state_ list is swapped so removed this state reset.

Reply:
Ya the planes will be left with previous state which will not be valid anymore, hence I think it's good to reset the state here.

Reply:
no more used then why to reset the state not going to impact anything. I tested also